apis/tests.py

```diff
@@ -115,5 +115,60 @@ def test_should_accept_filter_on_zipcode_and_list_gardens(self):
             zipcode_list.append(res["zipcode"])
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(json_response["count"], 2)
-        self.assertNotIn("93100", zipcode_list)
+        self.assertEqual(json_response['count'], 2)
+        self.assertNotIn('93100', zipcode_list)
+
+class TestGetGardenDetails(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create_user('hello@world', 'hello_world_123', has_garden=True)
+        self.garden = Garden.objects.create(user_id=User(id=self.user.id), description='', title='Monjardin', zipcode='75001')
+    
+
+    def test_should_get_garden_details_by_id(self):
+        response = self.client.get(f'/api/gardens/{self.garden.id}')
+        json_response = json.loads(response.content)
+        self.assertEqual(json_response['id'], self.garden.id)
+        self.assertEqual(json_response['description'],'')
```

Comment:
it's better to fetch the description info from the garden.

for example: `self.assertEqual(json_response["description"], self.garden.description)`
same for title & zipcode.